bitmap-team0/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const TB = require('./lib/TransformableBitmap.js');
+var ourTB = new TB('../img/palette-bitmap.bmp');
+
+ourTB.invert().grayscale().colorScale(1, 1.3, 0.8).write('out.bmp');
+console.log('chained transform written to out.bmp');
+
+const chooseTransform = require('./lib/cli.js');
```

Comment:
@bazacko @bhavyaab @ShaAllFar good inclusion of the `chooseTransform` module in your index file - that said, not sure if you should include the `ourTB` module in this file, with the associated transforms - this should be abstracted out and the user should be able to perform transforms through the CLI or by simply running the app and the appropriate methods being called (just not in the `index` file)